documentdb-playground/fleet-add-region/documentdb-resource-crp.yaml

```diff
@@ -0,0 +1,85 @@
+# Namespace definition
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: documentdb-preview-ns
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: documentdb-credentials
+  namespace: documentdb-preview-ns
+type: Opaque
+stringData:
+  username: default_user
+  password: {{DOCUMENTDB_PASSWORD}}
+
+---
+
+apiVersion: db.microsoft.com/preview
+kind: DocumentDB
+metadata:
+  name: documentdb-preview
+  namespace: documentdb-preview-ns
+spec:
+  nodeCount: 1
+  instancesPerNode: 1
+  documentDBImage: ghcr.io/microsoft/documentdb/documentdb-local:16
+  gatewayImage: ghcr.io/microsoft/documentdb/documentdb-local:16
+  resource:
+    storage:
+      pvcSize: 10Gi
+  environment: aks
+  clusterReplication:
+    highAvailability: true
+    crossCloudNetworkingStrategy: AzureFleet
+    primary: {{PRIMARY_CLUSTER}}
+    clusterList:
+{{CLUSTER_LIST}}
+  exposeViaService:
+    serviceType: LoadBalancer
+  logLevel: info
+
+---
+
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ClusterResourcePlacement
+metadata:
+  name: documentdb-namespace-crp
+spec:
+  resourceSelectors:
+    - group: ""
+      version: v1
+      kind: Namespace
+      name: documentdb-preview-ns
+      selectionScope: NamespaceOnly # only namespace itself is placed, no resources within the namespace
+  policy:
+    placementType: PickAll
+  strategy:
+    type: RollingUpdate
+
+---
+# Application team manages resources using RP
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ResourcePlacement
+metadata:
+  name: documentdb-resource-rp
+  namespace: documentdb-preview-ns
+spec:
+  resourceSelectors:
+    - group: db.microsoft.com
```

Comment:
Need to rebase. We have merged the new documentdb.io API. db.microsoft.com is no more available.